opus/import/import_for_tests.sh

```
@@ -19,7 +19,7 @@ python main_opus_import.py --do-all-import COISS_2002,COISS_2008,COISS_2111 $1
 python main_opus_import.py --do-all-import COUVIS_0002 $1
 python main_opus_import.py --do-all-import COVIMS_0006 $1
 python main_opus_import.py --do-all-import COCIRS_5408 $1
-python main_opus_import.py --import-check-duplicate-id --do-all-import GO_0016,GO_0017,GO_0018 $1
+python main_opus_import.py --import-check-duplicate-id --do-all-import GO_0016,GO_0017,GO_0018 --override-pds3-data-dir /data/pdsdata-gossi/holdings $1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)